fastlane/lib/fastlane/setup/setup_ios.rb

```diff
@@ -21,9 +21,11 @@ def run(user: nil)
       FastlaneCore::FastlaneFolder.create_folder! unless Helper.is_test?
       is_manual_setup = false
 
+      setup_project
+      react_native_pre_checks
+      ask_for_apple_id
```

Comment:
Because we don't access iTunes Connect or the Dev Portal at this point yet, so it's better ot not be inside the `begin`, see also the PR body 👍